File: packages/authentication/ios/Plugin/Classes/ConfirmVerificationCodeOptions.swift
import Foundation

@objc class ConfirmVerificationCodeOptions: NSObject {
    private var verificationId: String
    private var verificationCode: String

    init(verificationId: String, verificationCode: String) {
        self.verificationId = verificationId
        self.verificationCode = verificationCode
    }

    func getVerificationId() -> String {
        return verificationId
    }

    func getVerificationCode() -> String {
        return verificationCode
    }
}

File: packages/authentication/android/src/main/java/io/capawesome/capacitorjs/plugins/firebase/authentication/classes/ConfirmVerificationCodeOptions.java
package io.capawesome.capacitorjs.plugins.firebase.authentication.classes;

public class ConfirmVerificationCodeOptions {

    private String verificationId;
    private String verificationCode;

    public ConfirmVerificationCodeOptions(String verificationId, String verificationCode) {
        this.verificationId = verificationId;
        this.verificationCode = verificationCode;
    }

    public String getVerificationId() {
        return verificationId;
    }

    public String getVerificationCode() {
        return verificationCode;
    }
}
